store/sqlite/guest_links.go

```diff
@@ -76,16 +78,18 @@ func (s *Store) InsertGuestLink(guestLink picoshare.GuestLink) error {
 			max_file_bytes,
 			max_file_uploads,
 			creation_time,
-			expiration_time
+			url_expiration_time,
+			file_expiration_time
 		)
-		VALUES (:id, :label, :max_file_bytes, :max_file_uploads, :creation_time, :expiration_time)
+		VALUES (:id, :label, :max_file_bytes, :max_file_uploads, :creation_time, :url_expiration_time, :file_expiration_time)
 	`,
 		sql.Named("id", guestLink.ID),
 		sql.Named("label", guestLink.Label),
 		sql.Named("max_file_bytes", guestLink.MaxFileBytes),
 		sql.Named("max_file_uploads", guestLink.MaxFileUploads),
 		sql.Named("creation_time", formatTime(time.Now())),
-		sql.Named("expiration_time", formatExpirationTime(guestLink.Expires))); err != nil {
+		sql.Named("url_expiration_time", formatExpirationTime(guestLink.UrlExpires)),
+		sql.Named("file_expiration_time", guestLink.FileLifetime.Duration().Hours())); err != nil {
```

Comment:
Can we have a `formatFileLifetime` function rather than inlining the logic here? And then can we name the corresponding function `parseFileLifetime`?

```go
func formatFileLifetime(lt picoshare.FileLifetime) string {
	return lt.Duration().String()
}

func parseFileLifetime(s string) (picoshare.FileLifetime, error) {
	d, err := time.ParseDuration(s)
	if err != nil {
		return picoshare.FileLifetime{}, err
	}
	return picoshare.NewFileLifetimeFromDuration(d), nil
}
```

That way, we're just doing standard serialization / deserialization of `time.Duration` and don't need any custom logic.